invenio_vocabularies/records/models.py

```diff
@@ -56,22 +56,22 @@ class VocabularyMetadata(db.Model, RecordMetadataBase):
 
 
 class VocabularySubtype(db.Model):
-    """Vocabulary subtype type model.
+    """Vocabulary subtype model.
```

Comment:
This entire model needs to be moved to Subjects contrib and renamed to ``SubjectScheme``. I've opened an issue: https://github.com/inveniosoftware/invenio-vocabularies/issues/69

Reply:
Per discussion in standup: keep but rename to VocabularyScheme. The idea is that we might use it for affiliations at least. Let's see!